### PR TITLE
Log sitemap files for packages to delete them on uninstallation

### PR DESCRIPTION
**Pull request by joshuaruesweg**

### CODE REVIEW

File: wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php [hunk 1]
```diff
@@ -207,7 +207,7 @@ public function execute()
             }
 
             if ($this->workerData['dataCount'] + $this->limit > self::SITEMAP_OBJECT_LIMIT) {
-                $this->finishSitemap($this->sitemapObjects[$this->workerData['sitemap']]->objectType . '_' . $this->workerData['sitemapLoopCount'] . '.xml');
+                $this->finishSitemap($this->sitemapObjects[$this->workerData['sitemap']]->objectType . '_' . $this->workerData['sitemapLoopCount'] . '.xml', $this->sitemapObjects[$this->workerData['sitemap']]->packageID);
```
Comment (dtdesign) [inline wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php:210]: Please use temporary variables, this deep access is barely readable.

File: wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php [hunk 2]
```diff
@@ -218,7 +218,7 @@ public function execute()
             // finish sitemap
             if (\count($objectList) < $this->limit) {
                 if ($this->workerData['dataCount'] > 0) {
-                    $this->finishSitemap($this->sitemapObjects[$this->workerData['sitemap']]->objectType . '.xml');
+                    $this->finishSitemap($this->sitemapObjects[$this->workerData['sitemap']]->objectType . '.xml', $this->sitemapObjects[$this->workerData['sitemap']]->packageID);
```
Comment (dtdesign) [inline wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php:221]: Same as for line 210, use temporary variables please.

**Review by dtdesign — CHANGES_REQUESTED** [review]
LGTM

**Comment by joshuaruesweg** [comment]
Check failed, because `master` is failing atm.

**Comment by dtdesign** [comment]
> Check failed, because `master` is failing atm.

Already resolved.

**Review by TimWolla — APPROVED** [review]

**Pull request merged**